src/editors/containers/VideoEditor/components/VideoSettingsModal/components/DurationWidget.jsx

```diff
@@ -3,50 +3,85 @@ import { useDispatch } from 'react-redux';
 // import PropTypes from 'prop-types';
 
 import {
+  Col,
   FormControl,
   FormGroup,
+  Row,
 } from '@edx/paragon';
 
+import { injectIntl, intlShape, FormattedMessage } from '@edx/frontend-platform/i18n';
 import { keyStore } from '../../../../../utils';
 import CollapsibleFormWidget from './CollapsibleFormWidget';
 import hooks from './hooks';
+import { durationFromValue } from './duration';
+import messages from './messages';
 
 /**
  * Collapsible Form widget controlling video start and end times
  * Also displays the total run time of the video.
  */
-export const DurationWidget = () => {
+export const DurationWidget = ({
+  // injected
+  intl,
+}) => {
   const dispatch = useDispatch();
   const { duration } = hooks.widgetValues({
     dispatch,
     fields: { [hooks.selectorKeys.duration]: hooks.durationWidget },
   });
   const timeKeys = keyStore(duration.formValue);
+
+  const getTotalLabel = (startTime, stopTime) => {
+    if (!stopTime) {
+      if (!startTime) {
+        return intl.formatMessage(messages.fullVideoLength);
+      }
+      return intl.formatMessage(messages.startsAt, { startTime: durationFromValue(startTime) });
+    }
+    const total = stopTime - (startTime || 0);
+    return intl.formatMessage(messages.total, { total: durationFromValue(total) });
+  };
+
   return (
-    <CollapsibleFormWidget title="Duration">
-      <FormGroup size="sm">
-        <div>
+    <CollapsibleFormWidget
+      title={intl.formatMessage(messages.durationTitle)}
+      subtitle={getTotalLabel(duration.formValue.startTime, duration.formValue.stopTime)}
+    >
+      <FormattedMessage {...messages.durationDescription} />
+      <Row>
```

Comment:
Add top padding to prevent overlap between duration description and the start time floating label

Reply:
is there a paragon standard class for adding space like this?

Reply:
i think you can add to row `className="mt3"` and that should do it

Reply:
mt-4, in fact (applied to the Row below)